src/main/java/com/github/princesslana/slothbot/commands/ConfigurationCommand.java

```diff
@@ -1,13 +1,6 @@
 package com.github.princesslana.slothbot.commands;
 
-import com.github.princesslana.slothbot.Channel;
-import com.github.princesslana.slothbot.Config;
-import com.github.princesslana.slothbot.Discord;
-import com.github.princesslana.slothbot.Embed;
-import com.github.princesslana.slothbot.Limiter;
-import com.github.princesslana.slothbot.Moderator;
-import com.github.princesslana.slothbot.Rate;
-import com.github.princesslana.slothbot.Self;
+import com.github.princesslana.slothbot.*;
```

Comment:
No star imports

Reply:
My IDE did that for some reason, I'll fix it.